.github/workflows/golangci-lint.yml

```diff
@@ -12,9 +12,10 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/setup-go@v3
```

Comment:
I updated this as go 1.18 broke the former

See https://github.com/golangci/golangci-lint-action